presets/workspace/models/supported_models.yaml

```
@@ -6,14 +6,12 @@ models:
     tag: 0.0.1
 
   # Llama
-  # TODO: Uncomment when ready to trigger llama3 build process
-  # - name: llama3.1-8b-instruct
-  #   type: text-generation
-  #   version: https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct/commit/0e9e39f249a16976918f6564b8830bc894c89659
-  #   runtime: tfs
-  #   downloadAtRuntime: true
-  #   tag: 0.0.1
-
+  - name: llama3.1-8b-instruct
```

In https://github.com/kaito-project/kaito/blob/main/examples/inference/kaito_workspace_llama-3.1_8b_instruct.yaml#L12, the preset name is `llama-3.1-8b-instruct`. Could we add `-` after `llama` for consistency?

```suggestion
  - name: llama-3.1-8b-instruct
```